### PR TITLE
virtcontainers: Replace glog by logrus logger

### PR DESCRIPTION
**Pull request by sboeuf**
Glog causes some data race with project using our library and glog at the same time. For that reason, it is safer to move to logrus logger, which is more flexible and which does not cause such issues.

### CODE REVIEW

**Comment by sboeuf** [comment]
@sameo @dlespiau please take a look because we need this to avoid go test -race to fail in the proxy.

File: qemu.go [hunk 1]
```diff
@@ -76,19 +76,23 @@ const (
 type qmpGlogLogger struct{}
```
Comment (jodh-intel) [inline qemu.go]: Shouldn't this be renamed to `qmpLogger` or similar now?

**Comment by sameo** [comment]
LGTM, except for the qemu logger renaming.

**Comment by dlespiau** [comment]
side notes:

-  we should promote more structured logging in virtcontainers instead of just free form messages
- it may be interesting to delegate the actual logging to the user of the virtcontainers library. Something like gokit's logging interface would help a log there.

**Pull request merged**